internal/node/node.go

```diff
@@ -195,19 +196,33 @@ func (n *Node) init() error {
 		err = n.loadNodeConfig()
 		if err != nil {
 			n.log.Error("Failed to load node configuration file", slog.Any("err", err), slog.String("config_path", n.nodeOpts.ConfigFilepath))
-			err = fmt.Errorf("failed to load node configuration file: %s", err)
```

Comment:
Why is this no longer an error here?

Reply:
This line was to give a more detailed error message in the context of the node, since the nested telemetry error may be generic.

Reply:
that line didn't do anything. The value of `err` was set but never used.